libs/partners/ollama/langchain_ollama/chat_models.py

```diff
@@ -66,7 +66,10 @@ def _get_tool_calls_from_response(
     """Get tool calls from ollama response."""
     tool_calls = []
     if "message" in response:
-        if "tool_calls" in response["message"]:
+        if (
+            "tool_calls" in response["message"]
+            and response["message"]["tool_calls"] is not None
+        ):
```

Comment:
```suggestion
        if response["message"]["tool_calls"]:
```

Reply:
The suggested code will fail on old versions of ollama which don't always populate the tool_calls field.

Reply:
Ah right! Sorry

Reply:
This is correct then :)